worker/agents/legalIntakeAgent.ts

```
@@ -0,0 +1,895 @@
+            toolResult = await handleRequestLawyerReview(parameters, env, teamId);
+            break;
+          case 'schedule_consultation':
+            toolResult = await handleScheduleConsultation(parameters, env, teamConfig);
+            break;
+          default:
+            console.warn(`❌ Unknown tool: ${toolName}`);
+            const errorEvent = `data: ${JSON.stringify({
+              type: 'error',
+              message: `Unknown tool: ${toolName}`
+            })}\n\n`;
+            controller.enqueue(new TextEncoder().encode(errorEvent));
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)